### PR TITLE
Feature organizing

### PR DESCRIPTION
**Pull request by Danielxburns**

### CODE REVIEW

File: client/src/carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+// === STYLES === //
+
+const Wrapper = styled.div`
+  width: 1280px;
+  overflow: hidden;
+`;
+
+const CarouselContainer = styled.div`
+  display: flex;
```
Comment (bkwon94) [inline client/src/carousel.jsx:18]: I like the use of flex here. Also, very good and clean code overall.

File: client/src/carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  order: ${(props) => props.order};
+`;
+
+const Title = styled.div`
+  font-family: Alternate Gothic No1 D;
+  font-size: 20px;
+  color: #4c4c4c;
+`;
+
+const Role = styled.div`
```
Comment (bkwon94) [inline client/src/carousel.jsx]: I see all your styled components are divs. Is that what your component requires? I used many different elements from spans, ul's etc. Just something to look into.

File: client/src/carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      position: 0
+    };
+  }
+
+  getOrder(itemIndex) {
+    const { position } = this.state;
```
Comment (bkwon94) [inline client/src/carousel.jsx:79]: Object destructuring is great.

**Review by bkwon94 — APPROVED** [review]
Code is neat and organized. Excited to see how your service turns out later!

File: client/src/carousel.jsx (hunk 3 above)
Comment (Danielxburns) [reply client/src/carousel.jsx:79]: just now trying it out.  I was sticking with using the props keyword till I got it down.

File: client/src/carousel.jsx (hunk 2 above)
Comment (Danielxburns) [reply client/src/carousel.jsx]: This is a good point. I've been using divs as a default to limit the complexity while trying new things. I'll look into what might be more appropriate when I refactor.

File: client/src/carousel.jsx (hunk 1 above)
Comment (Danielxburns) [reply client/src/carousel.jsx:18]: Thanks!

**Pull request merged**